### PR TITLE
Remove Rails 5.2 support

### PR DESCRIPTION
**Pull request by dougmrqs**
Removes Rails 5.2 support due to EOL.

Referent to #1513

### CODE REVIEW

**Comment by HeitorMC** [comment]
> Hi again, @dougmrqs, would be possible to fix the conflicts in this PR? Thanks for this contribution.

Hey @VSPPedro, I talked with @dougmrqs and fixed this small conflicts for him :handshake:

**Comment by VSPPedro** [comment]
Nice! Thanks again for working on this!

**Comment by VSPPedro** [comment]
I'll put both of you guys as a co-authored in the squash commit. Is that okay?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,7 +34,6 @@ jobs:
           - rails_7_0
           - rails_6_1
           - rails_6_0
-          - rails_5_2
```
[the remark this answers is not in the document]
Comment (VSPPedro) [reply .github/workflows/ci.yml:37]: OK. I see you made this change in the other PR. Once approved. I'll merge this one first.

[the remark this answers is not in the document]
Comment (VSPPedro) [reply .github/workflows/ci.yml:37]: I received some pings in some PRs. Sorry. I did not have the time to take another look. I'll take a look as soon as possible.

**Review by mcmire — APPROVED** [review]
LGTM!

**Pull request merged**